modules/jaeger-deploy-production-es.adoc

```diff
@@ -0,0 +1,143 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-deploying.adoc
+////
+
+[id="jaeger-deploy-production_{context}"]
+= Deploying the Jaeger production strategy
+
```

Comment:
The ordering of the sections appears wrong - shouldn't the "Deploying the Jaeger X strategy" sections all be together, possibly with subsections for the different strategies?